tests/providers/amazon/aws/operators/test_ecs.py

```diff
@@ -349,6 +349,111 @@ def test_reattach_successful(self, launch_type, tags, start_mock, check_mock, wa
         check_mock.assert_called_once_with()
         assert self.ecs.arn == 'arn:aws:ecs:us-east-1:012345678910:task/d8c67b3c-ac87-4ffe-a847-4785bc3a8b55'
 
+    @parameterized.expand(
+        [
+            ['EC2', None],
+            ['FARGATE', None],
+            ['EC2', {'testTagKey': 'testTagValue'}],
+            ['', {'testTagKey': 'testTagValue'}],
+        ]
+    )
+    @mock.patch.object(ECSOperator, "_xcom_del")
+    @mock.patch.object(
+        ECSOperator,
+        "xcom_pull",
+        return_value="arn:aws:ecs:us-east-1:012345678910:task/d8c67b3c-ac87-4ffe-a847-4785bc3a8b55",
+    )
+    @mock.patch.object(ECSOperator, '_wait_for_task_ended')
+    @mock.patch.object(ECSOperator, '_check_success_task')
+    @mock.patch.object(ECSOperator, '_start_task')
+    def test_reattach_successful(
```

Comment:
There is already a test with the same name on L349

Reply:
Thanks, I removed the first one as it was redundant with this new one, more complete